### ISSUE TITLE
Add post-install-job/pre-delete-job to generate/delete all primitives for helm chart

### ISSUE THREAD

**Issue by gyliu513** [kubernetes-sigs/federation-v2#474]
After PR #426 , all of the primitives are now generated by `kubefed2 federate enable`, so with helm chart, we need to introduce `post-install-job/pre-delete-job` to generate and delete those primitives.

An example of `post-install-job` can be referenced here https://github.com/helm/helm/blob/master/docs/examples/nginx/templates/post-install-job.yaml

/cc @marun

**Comment by marun** [comment]
Is deletion of a control plane via helm something new or was that happening before?

**Comment by gyliu513** [comment]
@marun Yes, this is the behavior from day 1, please refer to https://github.com/kubernetes-sigs/federation-v2/blob/master/charts/federation-v2/README.md#uninstalling-the-chart for detail.

The following command will delete the control plane.

```shell
$ helm delete --purge federation-v2
```

**Comment by marun** [comment]
So this shouldn't be required if yaml is used instead of kubefed2?
